test/integration/imagechange_buildtrigger_test.go

```diff
@@ -82,9 +113,7 @@ func TestSimpleImageChangeBuildTrigger(t *testing.T) {
 	}
 
 	event = <-watch2.ResultChan()
-	event = <-watch2.ResultChan()
-
-	updatedConfig, err := openshift.Client.BuildConfigs(testNamespace).Get(config.Name)
+	updatedConfig, err = openshift.Client.BuildConfigs(testNamespace).Get(config.Name)
 	if err != nil {
 		t.Fatalf("Couldn't get BuildConfig: %v", err)
 	}
```

Comment:
as discussed on irc, the check is there for both buildconfigs. (ref-1 and ref-2)